kubernetes/spack/internal-spack-io/gh-gl-sync/cron-jobs.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: gh-gl-sync
+  namespace: spack
+spec:
+  schedule: "*/1 * * * *"
```

Comment:
This may take a bit of experimentation but every minute may be too frequent for spack given the sheer volume of PRs.

Reply:
Good point, Chuck. I think Todd initially asked for 1 minute resolution so people don't get impatient waiting for their PRs to get tested. If you're feeling brave Omar, I'm okay with trying this as-is and scaling it back if it proves to be problematic. 

Medium term, I believe the plan is to migrate this functionality to a GitHub app. At that point the CronJob would go away and the polling interval would be irrelevant.